src/Sylius/Bundle/AttributeBundle/Validator/Constraints/ValidDatetimeAttributeConfigurationValidator.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AttributeBundle\Validator\Constraints;
+
+use Sylius\Component\Attribute\AttributeType\DatetimeAttributeType;
+use Sylius\Component\Attribute\Model\AttributeInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Webmozart\Assert\Assert;
+
+final class ValidDatetimeAttributeConfigurationValidator extends ConstraintValidator
```

Comment:
Do we even need this one? It's the same logic as the date one, the only difference is the attribute type